fix_icon.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/src/gpx-viewer.c b/src/gpx-viewer.c
+index 11ef072..5419e39 100644
+--- a/src/gpx-viewer.c
++++ b/src/gpx-viewer.c
+@@ -225,7 +225,7 @@ void about_menuitem_activate_cb(void)
+     /* TODO: Add translators here */
+     gtk_show_about_dialog (NULL,
+         "program-name",     PACKAGE_NAME,
+-        "logo-icon-name",   "gpx-viewer",
++        "logo-icon-name",   "nl.sarine.gpx-viewer",
```

Comment:
you could just avoid the patch by setting `copy-icon: true` in the manifest.

Reply:
Thanks. Resolved.